src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -0,0 +1,352 @@
+
+```javascript
+const Sentry = require("@sentry/node");
+const Apm = require("@sentry/apm"); // This is required since it patches functions on the hub
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracesSampleRate: 0.1
+});
+```
+
+**Automatic Instrumentation**
+
+It’s possible to add tracing to all popular frameworks; however, we provide pre-written handlers only for Express.js.
+
+```javascript
+const Sentry = require("@sentry/node");
+const express = require("express");
+const app = express();
+
+// RequestHandler creates a separate execution-context using domains, so that every transaction/span/breadcrumb has it's own Hub to be attached to
+app.use(Sentry.Handlers.requestHandler());
+// TracingHandler creates a tracing for every incoming request
+app.use(Sentry.Handlers.tracingHandler());
+
+// the rest of your app
+
+app.use(Sentry.Handlers.errorHandler());
+app.listen(3000);
+```
+
+[Spans]({%- link _documentation/performance/performance-glossary.md -%}#span) are instrumented for the following operations within a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction):
+
+- HTTP requests made with `request` or `get` calls using native `http` and `https` modules
+- Middlewares (Express.js only)
+
+To enable them:
+
+```bash
+$ npm install @sentry/integrations@{% sdk_version sentry.javascript.browser %}
+```
+
+```javascript
+const Sentry = require("@sentry/node");
+const Integrations = require("@sentry/integrations");
+const express = require("express");
+const app = express();
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracesSampleRate: 0.1,
+  integrations: [
+    // enable HTTP calls tracing
+    new Sentry.Integrations.Http({ tracing: true }),
+    // enable Express.js middleware tracing
+    new Integrations.Express({ app })
+  ],
+});
+```
+
+**Managing Transactions**
+
+Let’s say you want to create a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) for an expensive operation (for example,  `processItem`) and send the result to Sentry:
+
+```javascript
+app.use(function processItems(req, res, next) {
+  const item = getFromQueue();
+  const transaction = Sentry.getCurrentHub().startSpan({ op: "task",  transaction: item.getTransaction() })
+  
+  // processItem may create more spans internally (see next examples)
+  processItem(item, transaction).then(() => {
+    transaction.finish();
+    next();
+  })
+});
+```
+
+**Adding Additional Spans to the Transaction**
+
+The next example is called in the `processItem` function from the code snippet above. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind; each individual span also needs to be finished; otherwise, it will not show up in the `transaction`.
```

Comment:
```suggestion
The next example is called in the `processItem` function from the code snippet above. Our SDK can determine if there is a currently open `transaction` and add all newly created [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind; each individual span also needs to be finished; otherwise, it will not show up in the `transaction`.
```